### PR TITLE
FIX: don't show dashboard warning when s3 settings are globally confi…

### PR DESCRIPTION
**Pull request by OsamaSayegh**
…gured (2)

### CODE REVIEW

**Comment by Giacomino1** [comment]
I closed my account and then I had to close all the trheads out of kindness otherwise I would have to trash the emails and then I hope that this explanation is enough. Thank you!!!

Simone Bartolini

________________________________

**Comment by Giacomino1** [comment]
I closed my account and then I had to close all the trheads out of kindness otherwise I would have to trash the emails and then I hope that this explanation is enough. Thank you!!!

Simone Bartolini

________________________________

File: config/discourse_defaults.conf [hunk 1]
```diff
@@ -174,6 +174,9 @@ s3_access_key_id =
 s3_secret_access_key =
 s3_use_iam_profile = false
 s3_cdn_url =
+s3_backup_bucket =
```
Comment (SamSaffron) [inline config/discourse_defaults.conf]: we don't need to add these though... enable_s3_uploads is implied if it is specified via globals

Comment (OsamaSayegh) [reply config/discourse_defaults.conf]: Yes, but it looks like if a site doesn't specify those settings in globals then `s3_config_check` is going blow up with method undefined errors. Also tests are failing with the same error if I remove those.

**Comment by SamSaffron** [comment]
OK, I went with this simpler fix for now, yeah this is hairy code. 

https://github.com/discourse/discourse/commit/330e848d4a0f8ff867ef8c52901f33b10a4ece63

**Pull request closed without merging**